src/pages/consequences.module.scss

```diff
@@ -0,0 +1,56 @@
+@import '../assets/shared-styles/colors.module.scss';
+
+.riskBuying {
+    display: inline-block;
+    font-family: cera-light;
+    font-size:180%;
+    width:300px;
+}
+
+.offenderRegistry {
+    display: inline-block;
+    font-family: cera-bold;
+    font-size: 20px;
+    width: 300px;
```

Comment:
Remove the `width` properties from all the text areas--it makes it artificially limited on non-mobile sizes.

Reply:
Removed width properties